app/Services/Manager.php

```diff
@@ -37,9 +47,9 @@ public function exportCSV($model, $reportbacks = false)
         $users = $model->users;
 
         $headers = ['northstar_id', 'first_name', 'last_name', 'email', 'cell'];
+
         if ($reportbacks) {
```

Comment:
Is it meant to be a single Reportback with multiple items? If so, maybe let's not pluralize this variable?